rando_modules/logic.py

```diff
@@ -204,41 +199,18 @@ def _find_new_nodes_and_edges(
             )
             found_new_items = found_new_items or found_additional_items
         non_traversable_edges = non_traversable_edges_cpy.copy()
-        #logging.debug("non_traversable_edges_cpy after %s", non_traversable_edges_cpy)
-        #logging.debug("non_traversable_edges after %s", non_traversable_edges)
-
-        # Check if an item node is reachable which already has an item placed.
-        # Since nodes are usually removed from this dict the moment an item is
-        # placed there, this can only be true for node which's item is not being
-        # randomized (caused by turned off shop shuffle, coin shuffle or
-        # hidden panel shuffle).
-        # If this is the case add item to inventory and remove node
-        for node_id, item_node in reachable_item_nodes.items():
```

Comment:
What was that loop not needed at all for the longest time..?

Reply:
This code was used in the old forward fill algorithm. The function for writing the spoiler log used this function as well, but didn't use several of the local variables, so the loop wasn't actually needed.
During the switch from forward fill to assumed fill the actual reason for this code being here was dropped, but this function was never refactored to remove the unused code.